massa-grpc/src/lib.rs

```diff
@@ -36,3 +40,78 @@ pub mod stream;
 
 #[cfg(test)]
 mod tests;
+
+/// Slot range type
+#[derive(Clone, Debug, Default, Eq, Hash, PartialEq)]
+pub struct SlotRange {
+    // Start lot
+    start_slot: Option<Slot>,
+    // End slot
+    end_slot: Option<Slot>,
+}
+
+impl SlotRange {
+    /// Check if the slot range is valid
+    pub fn check(&self) -> Result<(), GrpcError> {
+        match (self.start_slot, self.end_slot) {
```

Comment:
missing e < s

Reply:
I'll make a deep tests on it today. Sorry